pages/implementation/connect-inspector-to-segment.mdx

```diff
@@ -0,0 +1,227 @@
+---
+layout: 'guide'
+title: 'Send data from Segment to Avo Inspector (no code needed)'
+---
+
+import Link from '../../components/Link';
+
+# Enabling Avo Inspector without code changes
+
+## Use Avo Inspector as Segment destination
+
+If you are sending your data to Segment you can route it to Avo Inspector without adding any code to your codebase.
+
+You'll need to create a Segment Destination powered by Segment Function with a code snippet provided in your Avo workspace for each app and environment combination.
+
+In Segment you usually have different sources for each app environment, for example if you have a single web app, you probably have Web App Dev and Web App Production sources in Segment.
+
+Unlike Segment, Avo sources contain environments in them, so you will only need to create an Avo source per app, in our example you'll need a single Avo source called Web App.
+
+After connecting your Segment Source to corresponding code snippet the event schemas of all events seen in Segment will automatically be sent to Avo Inspector and you'll be able to monitor the quality of your tracking implementation without any changes in your code!
+
+> Remember that production data takes up to 2 hours to appear in the Inspector dashboard
+
+> Development data is available without any delay
+
+## Diagram of the data flow when sending directly to Segment
+
+<center>
+  <img
+    height="500"
+    src={require('../../images/chart-segment-to-inspector.png')}
+    alt="Data flow from Segment to Avo Inspector"
+  />
+</center>
+
+## Diagram of the data flow when sending to Segment with Avo generated code
+
+<center>
+  <img
+    height="500"
```

Comment:
This image looks skewed on my screen. Does removing the `height` attribute fix it?